### DISCUSSION TITLE
Can't get SSL CA file to work with config

### DISCUSSION THREAD

**Discussion by pbrunier** [alexandrainst/node-red-contrib-postgresql#81] [Q&A] [answered]
Hi, 
I have a self-signed CA and want to load the CA file into the config of this node.
Now have it working with  {"rejectUnauthorized": false} in the SSL options, but whatever I do, I don't get the ca file to be accepted. Doesn't seem to do anything.

Tried:
{
    "rejectUnauthorized": true and false,
    "ca": "/path/to/file"
 }
-AND-
{
    "rejectUnauthorized": true and false,
    "ca": "COMPLETE_CA_CERT_STRING"
 }

When rejectUnauthorized is set to false it always works. When set to true it doesn't work.
The only message that shows up is: Error: self-signed certificate in certificate chain
I don't get an error stating that the CA file cannot be found or is invalid, suspecting that that part of the config isn't used.

Version:  0.14.0

Thanks in advance for your help.

**Comment by evan-neres** [comment, marked as answer]
You can insert the certificate using JSON
![image](https://github.com/user-attachments/assets/0334a69b-bd77-4bd5-ba88-8e0b53517a66)

It is important that you insert the /n tag so that the node understands the beginning and end of the certificate.
![image](https://github.com/user-attachments/assets/c92e61b5-7816-455d-99c7-44c499fd5c12)

**Comment by pbrunier** [comment]
Thank you!
